### DISCUSSION TITLE
Make an always visible horizontal scrolling bar for Alert & Hunt

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#2057] [Ideas] [answered]
Right now, we need to scroll down to the bottom to get the horizontal scrolling bar which is very impractical.
Please make an option to have the horizontal scrolling bar follow the vertical scroll.

**Comment by dougburks** [comment, marked as answer]
What is your screen resolution?

Have you tried collapsing the left sidebar to give more space to the main sections of Alert and Hunt?

**Reply by 0nnyx** [reply]
QHD monitor. Everything fits when fullscreen but am used to put 2 windows side by side forcing horizontal scroll even with sidebar collapsed. Optional but would be a nice to have imo.

**Reply by dougburks** [reply]
Hmm...I think you're probably better off avoiding the horizontal scroll bar altogether by either alt-tabbing between full screen windows or using a second monitor if you really need them side by side.
